### DISCUSSION TITLE
Welcome to presidio Discussions!

### DISCUSSION THREAD

**Discussion by omri374** [microsoft/presidio#385] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 Feel free to share questions or just thoughts in these discussions instead of issues (up to you).

**Comment by ghost** [comment]
Hi omri374, We're developing an anonymizer with the Dutch language. I was wondering how long it would take(or if it is even possible) for Presidio to 'learn' this language. Results from the demo show that to much is being anonymized right now. Can you give me your thoughts about this? 

With Kind Regards,

leroygabrielse

**Reply by omri374** [reply]
Hi @leroygab95, thanks for starting the discussion!

The default version of presidio works mainly with English. To adapt Presidio to a new language, there are two main things to take care of:
1. Use a different NLP model. Presidio supports both spaCy and Stanza models. See [this documentation](https://github.com/microsoft/presidio/blob/master/docs/tutorial_analyzer.md#using-multiple-languages) on how to use a different model
2. Set up the recognizers to fit a new language. If you plan to use the predefined ones already built in, the only thing you probably need to do is to use different context words. See [here an example](https://github.com/microsoft/presidio/blob/6fdf19ffaa81bb858dab477d378a644c08f9a328/presidio-analyzer/presidio_analyzer/predefined_recognizers/us_passport_recognizer.py#L14) on the context words for the passport recognizer
3. There might be a need to change the list of predefined recognizers on the recognizer registry object (see [here](https://github.com/microsoft/presidio/blob/7935adf03c80a0ba5d53c6affb7d7c78198affaa/presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py#L56)). This can also be done dynamically after instantiating the object.

Some additional information on developing or adapting recognizers can be found [here](https://github.com/microsoft/presidio/blob/master/docs/developing_recognizers.md).

Hope this is enough to get you started! Feel free to continue the discussion if you have more questions or issues.
